### ISSUE TITLE
environment error: cannot run demo.py 

### ISSUE THREAD

**Issue by zipxup** [Zielon/MICA#6]
Hi, 
I've followed the instruction to install the conda environment, but I cannot run demo.py successfully, this is the error:

> ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject

i searched online and upgraded numpy from 1.21.2 to 1.23.1 by using `pip install -U numpy`

Then I ran into another error:

> ImportError: numpy.core.multiarray failed to import
 
It is due to version incompatibility, but which one is the correct version? Can you please help me?

**Comment by Zielon** [comment]
Did you use the `install.sh` script? I am using those ones:
```
  - numpy=1.21.2=py38h20f2e39_0
  - numpy-base=1.21.2=py38h79a1101_0
  ```

**Comment by zipxup** [comment]
yes, numpy version was 1.21.2 after i used `install.sh`, but i ran into this error:

> ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject

**Comment by zipxup** [comment]
what is your c compiler's version?

**Comment by sumeromer** [comment]
I could install a conda environment from the environment.yaml, however, got exactly the same error. Later, I noticed it is related to NumPy and Cython (some parts of insightface). Inside the same environment, I reinstalled NumPy and later upgraded Scipy as it was not compatible with the newer version of NumPy. (numpy==1.23.3, scipy==1.9.1).
```bash
pip install --upgrade numpy
pip install --upgrade scipy
```
Later I deleted the environment and included those chances in yaml file:
I changed the NumPy version in those lines (and removed SciPy among the conda packages):
```
  - numpy=1.23.1
  - numpy-base=1.23.1
  - pip:
    - scipy==1.9.1
```
I guess something in the dependencies requires scipy==1.7.3, but everything works in the newer version, too. Now, the demo file is working w/o any warning or error.
